code/__HELPERS/tool_helpers.dm

```diff
@@ -36,7 +36,7 @@
 	return FALSE
 
 /proc/ispowertool(O)//used to check if a tool can force powered doors
```

Comment:
Interesting that not everything under `/obj/item/powertool` counts as "power tool"...
(Not something you can really fix, just an observation).

Reply:
Might be better to have this as an item level proc and just return TRUE if the item is a powertool